azure-pipelines.yml

```diff
@@ -407,142 +407,131 @@ jobs:
     - bash: ./scripts/ci/test_profile_integration.sh
       displayName: 'Run Integration Test against Profiles'
 
-- job: RunAutomationPython3_6
-  displayName: Run Automation Python 3.6
+- job: AutomationTest
+  displayName: Automation Test (Profile Latest)
   dependsOn: BuildPythonWheel
-  timeoutInMinutes: 90
+  timeoutInMinutes: 120
```

Comment:
should it be incremental?

Reply:
pytest take more time to run all tests (1936), so, have to increase this timeout limit. And Will be fixed in the future to optimize the time cost.

Reply:
What's the time cost now?

Reply:
1H 49m from this [build ](https://dev.azure.com/azure-sdk/public/_build/results?buildId=370756&view=logs&j=74095127-2a27-5370-37ed-15a4193f243f&t=74095127-2a27-5370-37ed-15a4193f243f) from this PR https://github.com/Azure/azure-cli/pull/13179